diff --git a/wrangler-dist/cli.js b/wrangler-dist/cli.js
index de586b6..920d3f6 100644
--- a/wrangler-dist/cli.js
+++ b/wrangler-dist/cli.js
@@ -180693,7 +180693,10 @@ var init_dist14 = __esm({
             this.onPreviewTokenExpired();
             break;
           case "error":
-            this.emitErrorEvent("Error inside ProxyWorker", message.error);
+            logger4.error(
+              "Error proxying request to the local Worker:",
+              message.error.message
+            );
             break;
         }
       }
@@ -208016,7 +208019,10 @@ var init_ProxyController = __esm({
             this.emitPreviewTokenExpiredEvent(message.proxyData);
             break;
           case "error":
-            this.emitErrorEvent("Error inside ProxyWorker", message.error);
+            logger2.error(
+              "Error proxying request to the local Worker:",
+              message.error.message
+            );
             break;
           case "debug-log":
             logger2.debug("[ProxyWorker]", ...message.args);
